backend/ventserver/simulation.py

```diff
@@ -15,7 +15,7 @@
 try:
     from ventserver.io.trio import rotaryencoder
 except RuntimeError:
-    logging.getLogger().warning('Running without RPi.GPIO!')
+    logging.getLogger().warning('Running without pigpio!')
```

Comment:
Now that we're using pigpio instead of RPi.GPIO, is this exception handling on import still necessary, or can we run the script without it?

Reply:
Yeah, we don't need that anymore. I'm removing it for now, but we'll have to test it first so that it won't cause chaos again.